src/components/reusable/CurrentDate/CurrentDate.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+class CurrentDate extends React.Component {
+    today = null;
+    date = null;
+
+    constructor() {
+        super();
+        this.today = new Date();
```

Comment:
What if you run you App at 11 59: 59 PM ?  within a second new date should be rendered but I doubt that would happen in your case... ;)